tests/Phinx/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -402,6 +402,19 @@ public function testAddStringColumnWithSignedEqualsFalse()
         $this->assertEquals('varchar(255)', $rows[1]['Type']);
     }
 
+    public function testAddStringColumnWithCustonCollation()
+    {
+        $table = new \Phinx\Db\Table('table1', array('collation' => 'utf8_general_ci'), $this->adapter);
+        $table->save();
+        $this->assertFalse($table->hasColumn('string_collation_default'));
+        $this->assertFalse($table->hasColumn('string_collation_custom'));
+        $table->addColumn('string_collation_default', 'string', array())->save();
+        $table->addColumn('string_collation_custom', 'string', array('collation' => 'utf8mb4_unicode_ci'))->save();
+        $rows = $this->adapter->fetchAll('SHOW FULL COLUMNS FROM table1');
+        $this->assertEquals('utf8_general_ci', $rows[2]['Collation']);
```

Comment:
Just can't run these tests on my environment. Please, check them and fix is there are some issues with this test.